Mathlib/Combinatorics/SimpleGraph/Matching.lean

```diff
@@ -133,6 +133,21 @@ lemma IsMatching.coeSubgraph {G' : Subgraph G} {M : Subgraph G'.coe} (hM : M.IsM
   · obtain ⟨_, hw', hvw⟩ := (coeSubgraph_adj _ _ _).mp hy
     rw [← hw.2 ⟨y, hw'⟩ hvw]
 
+protected lemma IsMatching.map {G' : SimpleGraph W} {M : Subgraph G} (f : G →g G')
+    (hf : Injective f) (hM : M.IsMatching) : (M.map f).IsMatching := by
+  rintro _ ⟨v, hv, rfl⟩
+  obtain ⟨v', hv'⟩ := hM hv
+  use f v'
+  refine ⟨⟨v, v', hv'.1, rfl, rfl⟩, ?_⟩
+  rintro _ ⟨w, w', hw, hw', rfl⟩
+  cases hf hw'.symm
+  rw [hv'.2 w' hw]
+
+@[simp]
+lemma Iso.isMatching_map {G' : SimpleGraph W} {M : Subgraph G} (f : G ≃g G') :
+    (M.map f.toHom).IsMatching ↔ M.IsMatching where
+   mp h := by simpa [← map_comp] using h.map f.symm.toHom f.symm.injective
+   mpr := .map f.toHom f.injective
 /--
```

Comment:
```suggestion
   mpr := .map f.toHom f.injective

/--
```

Reply:
@adomani Could you take a look at what happened here? My suggestion [here](https://github.com/leanprover-community/mathlib4/pull/17436#discussion_r1832706753) contained an intentional new line, but the new line accidentally contained spaces. After that change was committed, the linter complained (correctly) about whitespaces on the line. But the fix was seemingly to remove the line, rather than to change it to an empty line (thus the change here ^). Is this the lint fixer being weird, or github suggestions being weird? In either case, is there a fix?

(@pimotte sorry for hijacking your PR, you're free to approve+merge it now!)